frontend/public/components/dashboard/project-dashboard/status-card.tsx

```diff
@@ -1,22 +1,64 @@
 import * as React from 'react';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector } from 'react-redux';
 import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
 import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
 import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
 import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
 import HealthBody from '@console/shared/src/components/dashboard/status-card/HealthBody';
 import { Status } from '@console/shared';
+import { RootState } from '@console/internal/redux';
+import { getActiveNamespace } from '@console/internal/reducers/ui';
+import {
+  DashboardsOverviewHealthResourceSubsystem,
+  DashboardsOverviewHealthSubsystem,
+  isDashboardsOverviewHealthResourceSubsystem,
+  isDashboardsOverviewHealthSubsystem,
+  useExtensions,
+} from '@console/plugin-sdk';
+import { ContainerSecurityFlag } from '@console/container-security/src/const';
```

Comment:
we should not import from `container-security` in public

Reply:
fixed